src/cljs/main/broadfcui/common.cljs

```diff
@@ -178,7 +178,7 @@
 
 (def set-type->membership-attribute
   {"participant_set" "participants"
-   "sample_set" "samples"
+   "sample_set" :samples
```

Comment:
I don't like this.  Leave this as a string, and convert to a keyword on line 203.